packages/webpack-pwa-manifest-plugin/tests/index.js

```diff
@@ -0,0 +1,5 @@
+const runTest = require('./runTest');
+const tests = ['basic'];
+
+console.log('Running tests...');
```

Comment:
Move this into the test runner function, since it's responsible for printing out the rest of the test status output